diff --git a/node_modules/@fastify/deepmerge/index.js b/node_modules/@fastify/deepmerge/index.js
index e3ba232..e733358 100644
--- a/node_modules/@fastify/deepmerge/index.js
+++ b/node_modules/@fastify/deepmerge/index.js
@@ -65,11 +65,11 @@ function deepmergeConstructor (options) {
     return result
   }
 
-  const getKeys = options?.symbols
+  const getKeys = (options && options.symbols)
     ? getSymbolsAndKeys
     : Object.keys
 
-  const cloneProtoObject = typeof options?.cloneProtoObject === 'function'
+  const cloneProtoObject = (options && typeof options.cloneProtoObject === 'function')
     ? options.cloneProtoObject
     : undefined
 
@@ -161,7 +161,7 @@ function deepmergeConstructor (options) {
     return result
   }
 
-  return options?.all
+  return (options && options.all)
     ? _deepmergeAll
     : _deepmerge
 }
